flutter/lib/src/sentry_flutter.dart

```diff
@@ -243,6 +245,19 @@ mixin SentryFlutter {
     options.sdk = sdk;
   }
 
+  static void _setRedactionOptions(SentryFlutterOptions options) {
+    if (options.experimental.privacy != null) {
+      return;
+    } else if (options.screenshot.attachScreenshot == true &&
+        !options.experimental.replay.isEnabled) {
+      options.experimental.privacy = SentryPrivacyOptions()
+        ..maskAllText = false
+        ..maskAllImages = false;
+    } else {
+      options.experimental.privacy = SentryPrivacyOptions();
+    }
+  }
```

Comment:
maybe also makes sense to add a short comment why we are doing it this way to make it a bit more clear